parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -165,6 +165,7 @@ configureSwapFile
 
 ensureSysctl
 ensureKubelet
+ensureLabels
```

Comment:
move this to after ensureJournal? Also give the kubelet a bit time to warm up

Reply:
After ensureJournal. Were you thinking give even more time than that, or is that good enough?

Reply:
I think it's good enough.